sdk/src/Services/APIGateway/Generated/Model/Internal/MarshallTransformations/ImportRestApiRequestMarshaller.cs

```diff
@@ -60,14 +60,17 @@ public IRequest Marshall(ImportRestApiRequest publicRequest)
             request.HttpMethod = "POST";
 
             request.AddSubResource("mode", "import");
-            
             if (publicRequest.IsSetFailOnWarnings())
-                request.Parameters.Add("failonwarnings", StringUtils.FromBool(publicRequest.FailOnWarnings));
-            
+            {
+                if(!request.Parameters.ContainsKey("failonwarnings"))
```

Comment:
I am confused how it can ever already exist and if so we don't add the value on the request. Can you clarify?

Reply:
Since the order of the properties being generated is not gauranteed, it is possible that the query param of maps inserted the key first, in which case we would have to check if the key already existed. This may not matter however, if you read my comment above